src/classes/Shapes/CompoundShape.js

```diff
@@ -19,9 +24,31 @@ Goblin.CompoundShape = function() {
  */
 Goblin.CompoundShape.prototype.addChildShape = function( shape, position, rotation ) {
 	this.child_shapes.push( new Goblin.CompoundShapeChild( shape, position, rotation ) );
+	this.updateCenterOfMass();
 	this.calculateLocalAABB( this.aabb );
 };
 
+Goblin.CompoundShape.prototype.updateCenterOfMass = function () {
+	var i;
+
+	if ( this.center_of_mass_override ) {
```

Comment:
this.center_of_mass_override seems to be always null.

Reply:
It's meant to be accessed from the outer code (by default it's null because physics engine never overrides center of mass).